apps/mobile/src/components/register-adoption-form/adoption-form.hooks.test.tsx

```diff
@@ -1,12 +1,15 @@
 import { act, renderHook } from '@testing-library/react-hooks';
+import { useToast } from 'native-base';
+import { useFormContext } from 'react-hook-form';
+import useLocale from '../../shared/hooks/use-locale';
 
-import { useMultiStepNavigation } from './adoption-form.hooks';
+import { useFormValidation, useMultiStepNavigation } from './adoption-form.hooks';
 import { AdoptionSteps } from './adoption-form.types';
 
 describe('useMultiStepNavigation', () => {
```

Comment:
Could you leave this file dedicated to the `useMultiStepNavigation` hook? Cause this is already too big, then you can create another one only for `useFormValidation`. Let's create one file/test file for each hook, to have better readability.